src/main/java/ch/njol/skript/conditions/CondIsStackable.java

```diff
@@ -0,0 +1,36 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+
+
+
+@Name("is Stackable")
+@Description("Check if item is stackable.")
+@Examples("%item% is stackable")
+@Since("2.3")
+
+public class CondIsStackable extends PropertyCondition<ItemType> {
+    static {
+        
```

Comment:
There is a blank line here.

Reply:
That empty line should be above static block not inside it